HC.Tomek/HC.Tomek.Js.Ex3/questionaireAsync.js

```diff
@@ -0,0 +1,43 @@
+var questionaire = questionaire || {};
+
+(function(){
+  var shuffle = function(tab) {
+      for (var i = tab.length; i; i--) {
+          var j = Math.floor(Math.random() * i);
+          [tab[i - 1], tab[j]] = [tab[j], tab[i - 1]];
```

Comment:
wydaje mi się że można to zrobić dużo prościej :) zadanie wydaje się przez to skomplikowane, a to tak naprawde zwykła tablica obiektów (pytań)